.github/workflows/deploy-prod.yml

```diff
@@ -0,0 +1,12 @@
+name: Deploy to Prod PyPi
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy-prod:
```

Comment:
Should we maybe only push if we create a tag ?

Reply:
Thought this might be a relatively low risk place to do continuous deployment